src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/EverythingOperationContinuationToken.cs

```diff
@@ -3,42 +3,88 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using System.Text.Json;
+using System.Collections.Generic;
+using Newtonsoft.Json;
 
 namespace Microsoft.Health.Fhir.Core.Features.Operations.Everything
 {
-    public class EverythingOperationContinuationToken
+    internal class EverythingOperationContinuationToken
     {
-        public EverythingOperationContinuationToken(int phase, string internalContinuationToken)
+        [JsonConstructor]
+        internal EverythingOperationContinuationToken(int phase, string internalContinuationToken)
```

Comment:
This constructor looks a bit confusing since some members are set by input variables while some others are set by default values. We can update this to `EverythingOperationContinuationToken()` and set `Phase` to 0 and `InternalContinuationToken` to null in the constructor. This is the only scenario in our code that it is called.

Reply:
Consistency is always nice 😊. Fixed in 16ec002.